vscode-lean4/package.json

```diff
@@ -278,6 +278,12 @@
 				"title": "Infoview: Toggle Expected Type",
 				"description": "Toggle the display of expected type."
 			},
+			{
+				"command": "lean4.infoview.goToDefinition",
```

Comment:
I hate how roundabout this implementation has to be because of VSCode's limitations. Is there a way to hide this command from users' views? If you run it from the command palette right now it just crashes, and even if it didn't it is guaranteed to be confusing (since, for example, it will not go-to-def for the token under the cursor which is a plausible interpretation of what it might do based on just the name).

Reply:
Yes, we can hide it with `"when": "false"`. I initially thought that not adding it to the command palette menu would be sufficient, but apparently VS Code includes all commands there regardless.